bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,22 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
-python:
-  - 2.7
+  - eggs
```

Comment:
you need to give it a value. Given the relative URL in the ``directories`` is the git clone path, this should do it:

```
directories:
  - buildout-cache

[...]

./bin/buildout [....] buildout:download-cache=${buildout:directory}/download-cache
```